src/metakb/transform/civic.py

```diff
@@ -898,7 +902,11 @@ def _add_eid_document(self, source: Dict) -> Optional[Document]:
             if document not in self.documents:
                 self.documents.append(document)
         else:
-            logger.debug("%s not in schemas.SourcePrefix", source_type)
+            logger.debug(
```

Comment:
Might be worth making this a higher warning level? I'd think of "debug" as more appropriate for cases like "this is a very granular detail about what is happening because something mysterious is breaking and I'm tracking the pipeline step by step", whereas this would be something that I'd want to know about even if the pipeline runs successfully